File: Php/codice/L07/model/Prodotto.php
<?php

class Prodotto {

    private $id;
    private $nome;
    private $categoria;
    private $prezzo;
    private $giacenza;



    public function __get($name){
        return $this->$name;
    }

    public function __set($name, $value){
        $this->$name = $value;
    }

    public function __tostring(){
        return $this->nome . ': ' . $this->prezzo;
    }


}

File: Java/carrello-spring/carrello-prodotti/src/main/java/com/maboglia/carrello_prodotti/entities/Prodotto.java
package com.maboglia.carrello_prodotti.entities;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;

@Entity
@Table(name = "prodotti")
@Data
public class Prodotto {

    @Id
    private int id;
    private String nome;
    private String categoria;
    private double prezzo;
    private int giacenza;

}
